src/Rules/Gedmo/PropertiesExtension.php

```diff
@@ -72,12 +72,19 @@ public function isInitialized(PropertyReflection $property, string $propertyName
 	 */
 	private function isGedmoAnnotationOrAttribute(PropertyReflection $property, string $propertyName, array $classList): bool
 	{
-		if ($this->annotationReader === null) {
+		$classReflection = $property->getDeclaringClass();
+		if ($this->objectMetadataResolver->isTransient($classReflection->getName())) {
 			return false;
 		}
 
-		$classReflection = $property->getDeclaringClass();
-		if ($this->objectMetadataResolver->isTransient($classReflection->getName())) {
+		$attributes = $propertyReflection->getAttributes();
```

Comment:
```suggestion
		$propertyReflection = $classReflection->getNativeReflection()->getProperty($propertyName);

		$attributes = $propertyReflection->getAttributes();
```

Need to move the property reflector up (then remove from [here](https://github.com/phpstan/phpstan-doctrine/pull/427/files#diff-1d1d864e71324b0342c2e8faec3c88eefd2673ae50572bec85b0dfb22b61e910R91))